snapshots/energy_institute/2023-06-26/statistical_review_of_world_energy.csv.dvc

```diff
@@ -0,0 +1,23 @@
+meta:
+  name: Energy Institute Statistical Review of World Energy 2023
+  publication_year: 2023
+
+  publication_date: '2023-06-26'
+  source_name: Energy Institute Statistical Review of World Energy 2023
```

Comment:
I think the year should be in brackets to match our usual format for the bottom left source.